tools/c7n_azure/c7n_azure/resources/storage.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_azure.query import QueryResourceManager
+from c7n_azure.provider import resources
+
+from c7n.actions import (
+    ActionRegistry, BaseAction
+)
+from c7n.filters import (
+    FilterRegistry, ValueFilter, Filter, OPERATORS
+)
+
+filters = FilterRegistry('azure.storage.filters')
```

Comment:
fwiw, the metaclass will add these to the query resource manager subclasses, so no need to define separately.